docs/FAQ.md

```diff
@@ -144,3 +144,23 @@ TigerBeetle](https://github.com/tigerbeetle/tigerbeetle#quickstart),
 [follow our
 development](https://github.com/tigerbeetle/tigerbeetle#short-term-roadmap),
 and [give feedback](https://join.slack.com/t/tigerbeetle/shared_invite/zt-1gf3qnvkz-GwkosudMCM3KGbGiSu87RQ)!
+
+## Is TigerBeetle ACID-compliant?
+
+Yes. Let's discuss each part:
+
+1. Atomicity: TigerBeetle allows events within a batch to be
+   [linked](./reference/transfers.md#flagslinked). When an event fails
+   within a linked chain of events, all events are rolled back.
+
+2. Consistency: TigerBeetle's state machine ensures that accounts and
+   transfers remain consistent. When you create a transfer, associated
+   account credits and debits are guaranteed to be up-to-date.
+
+3. Isolation: All events within a batch, across all clients, are
+   serialized into TigerBeetle's state machine. There are no
+   concurrent events in TigerBeetle, even across clients.
+
+4. Durability: TigerBeetle `fsync()`s data and replicates it among a
+   quorum of replicas in a cluster before indicating success to the
+   user.
```

Comment:
Marketing-wise, I think it would build trust if we specify the extent of durability. ACI are binary properties, while D is a slider (there's no durability if a sufficiently large asteroid hits the Earth). By showing that we know exactly where our D ends, we build trust. There's also angle that we tolerate storage faults, so that we again explicitly thinking through durability limits

We can convey  the following: 

- on a single replica, we use direct IO, so:
  - a replica only acknowledges transaction once the data hits the disk (fsync gives you this)
  - if a hardware error occurs, the replcia noticies it and doesn't ack (fsync _dosen't_ give you this)
- on a cluster level, a transaction is only acknowledge if a quorum of replicas acknoledge it
- once a cluster acks a transaction, it is never "lost", even if subsequently all those quorum replicas fail (the cluster would stop processing transactions when that happens, )
- after saving transaction to disk, we don't blidly assume that it is there, but take into account the possibility that the data gets corrupted on some subsequent read, and actively repair it. If at least one replcia has a comitted transaction, it will be repaired to the whole cluster. 


Not sure how to best compress that to a couple of pithy sentences, but I think the core idea we want to convey is:

> unlike literary everyone else, we know that disk fail, and the data we write today might not be available tomorrow. We actively work against such entropy by taking advantage of the cluster-wide storage: a bit of data should get corrupted on all machines to get lost, and even in that case the system would safely halt.

Reply:
Got it, that makes sense. I pushed another shot at it incorporating your suggestions.

Reply:
Hm, I'd say this needs some wordsmithing more, opening with "durability is a sliding scale" is a little bit too blunt I feel :) 

Maybe something like this "Absolute durability is impossible, because all hardware can ultimately fail. TigerBeetle embraces limited disk reliability and maximizes data durability in spite of imperfect disk. Data we write today might not be available tomorrow ....". 

And maybe more emphasis on TigerBeetle rather on we

Reply:
Cool, updated again. :)